### DISCUSSION TITLE
Thrust command

### DISCUSSION THREAD

**Discussion by fernandafenelon** [JSBSim-Team/jsbsim#538] [Q&A]
Good afternoon everyone,

Does anyone know how to increase the thrust during a simulation?
I want to do a thrust step to 1.2*Initial thrust.

**Comment by seanmcleod** [comment]
Take a look at the `B737_Runway.xml` script for an example:

https://github.com/JSBSim-Team/jsbsim/blob/01da4616c7e9c920216624806f15943441004bbc/scripts/B737_Runway.xml#L29-L30
